scvi/model/_jaxscvi.py

```diff
@@ -420,10 +143,12 @@ def get_latent_representation(
             adata=adata, indices=indices, batch_size=batch_size, iter_ndarray=True
         )
 
+        bound_module = self.module.bound_module
+
         @jax.jit
         def _get_val(array_dict):
```

Comment:
We should add this to the wrapper so that devs don't need to interact with the bound module in most cases. Also, Flax does not recommend using these bound modules extensively for some reason.

Reply:
sounds good